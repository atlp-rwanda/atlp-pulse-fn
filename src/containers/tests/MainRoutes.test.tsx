import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import MainRoutes from '../Routes';

describe('Main Routes', () => {
  it('Should render', () => {
    const elem = renderer
      .create(
        <MemoryRouter>
          <MainRoutes />
        </MemoryRouter>,
      )
      .toJSON();
    expect(elem).toMatchSnapshot();
  });
});
