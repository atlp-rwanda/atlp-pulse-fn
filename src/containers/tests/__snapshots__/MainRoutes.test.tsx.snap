// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Routes Should render 1`] = `
<div
  className="flex flex-col min-h-screen"
>
  <div
    className="w-screen h-[8vh] z-10 bg-white dark:bg-dark-bg fixed border-b undefined"
  >
    <div
      className="px-3 flex justify-between items-center w-full h-full"
    >
      <div
        className="flex items-center h-full justify-between lg:w-full"
      >
        <a
          className="flex flex-row lg:px-5"
          href="/"
          onClick={[Function]}
        >
          <img
            alt="logo"
            className="w-full cursor-pointer mr-2"
            src={Object {}}
          />
          <h1
            className="text-3xl font-bold font-lexend text-primary dark:text-dark-text-fill"
          >
            PULSE
          </h1>
        </a>
        <ul
          className="hidden lg:flex cursor-pointer"
        >
          <li
            className="px-5 text-xl dark:text-dark-text-fill"
          >
            <a
              aria-current="page"
              className="text-primary"
              href="/"
              onClick={[Function]}
            >
              Home
            </a>
          </li>
          <li
            className="px-5 text-xl dark:text-dark-text-fill"
          >
            <a
              className=""
              href="/pricing"
              onClick={[Function]}
            >
              Pricing
            </a>
          </li>
          <li
            className="px-5 text-xl dark:text-dark-text-fill"
          >
            <a
              className=""
              href="/product"
              onClick={[Function]}
            >
              Product
            </a>
          </li>
        </ul>
      </div>
      <div
        className="hidden lg:flex lg:w-full justify-end "
      >
        <button
          className="px-4 mt-1 cursor-pointer"
          id="theme-switch"
          onClick={[Function]}
          type="button"
        >
          <svg
            aria-hidden="true"
            className="w-8"
            fill="currentColor"
            viewBox="0 0 20 20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M17.293 13.293A8 8 0 016.707 2.707a8.001 8.001 0 1010.586 10.586z"
            />
          </svg>
        </button>
        <a
          href="/login/org"
          onClick={[Function]}
        >
          <button
            className="btn primary lg undefined"
            disabled={false}
            type="button"
          >
             
            Sign In
             
          </button>
        </a>
        <a
          href="/signup/org"
          onClick={[Function]}
        >
          <button
            className="btn transparentbtn lg mr-8"
            disabled={false}
            type="button"
          >
             
            Register an organization
             
          </button>
        </a>
      </div>
      <div
        className="flex px-5 lg:hidden"
      >
        <button
          className="px-3"
          onClick={[Function]}
          type="button"
        >
          <svg
            aria-hidden="true"
            className="w-6 mr-2"
            fill="currentColor"
            viewBox="0 0 20 20"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M17.293 13.293A8 8 0 016.707 2.707a8.001 8.001 0 1010.586 10.586z"
            />
          </svg>
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          <svg
            aria-hidden="true"
            className="w-7 dark:text-dark-text-fill"
            fill="none"
            stroke="currentColor"
            strokeWidth={2}
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M4 6h16M4 12h16M4 18h16"
              strokeLinecap="round"
              strokeLinejoin="round"
            />
          </svg>
        </button>
      </div>
    </div>
    <ul
      className="hidden"
    >
      <li
        className="p-2 w-full mt-2 dark:text-dark-text-fill text-primary"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li
        className="p-2 w-full dark:text-dark-text-fill"
      >
        <a
          href="/pricing"
          onClick={[Function]}
        >
          Pricing
        </a>
      </li>
      <li
        className="p-2 w-full dark:text-dark-text-fill"
      >
        <a
          className="w-full"
          href="/product"
          onClick={[Function]}
        >
          Product
        </a>
      </li>
      <li
        className="p-2 w-full dark:text-dark-text-fill mt-6 mb-2 bg-primary text-white rounded-md px-[35%]"
      >
        <a
          className="w-full"
          href="/login/org"
          onClick={[Function]}
        >
           
          Sign In
        </a>
      </li>
      <a
        href="/signup/org"
        onClick={[Function]}
      >
        <button
          className="btn transparentbtn lg mr-8"
          disabled={false}
          type="button"
        >
           
          Register an organization
           
        </button>
      </a>
    </ul>
  </div>
  <div
    className="hero"
  >
    <div
      className="w-screen h-[8vh] z-10 bg-white dark:bg-dark-bg fixed border-b bg-opacity-50 dark:bg-opacity-50"
    >
      <div
        className="px-3 flex justify-between items-center w-full h-full"
      >
        <div
          className="flex items-center h-full justify-between lg:w-full"
        >
          <a
            className="flex flex-row lg:px-5"
            href="/"
            onClick={[Function]}
          >
            <img
              alt="logo"
              className="w-full cursor-pointer mr-2"
              src={Object {}}
            />
            <h1
              className="text-3xl font-bold font-lexend text-primary dark:text-dark-text-fill"
            >
              PULSE
            </h1>
          </a>
          <ul
            className="hidden lg:flex cursor-pointer"
          >
            <li
              className="px-5 text-xl dark:text-dark-text-fill"
            >
              <a
                aria-current="page"
                className="text-primary"
                href="/"
                onClick={[Function]}
              >
                Home
              </a>
            </li>
            <li
              className="px-5 text-xl dark:text-dark-text-fill"
            >
              <a
                className=""
                href="/pricing"
                onClick={[Function]}
              >
                Pricing
              </a>
            </li>
            <li
              className="px-5 text-xl dark:text-dark-text-fill"
            >
              <a
                className=""
                href="/product"
                onClick={[Function]}
              >
                Product
              </a>
            </li>
          </ul>
        </div>
        <div
          className="hidden lg:flex lg:w-full justify-end "
        >
          <button
            className="px-4 mt-1 cursor-pointer"
            id="theme-switch"
            onClick={[Function]}
            type="button"
          >
            <svg
              aria-hidden="true"
              className="w-8"
              fill="currentColor"
              viewBox="0 0 20 20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M17.293 13.293A8 8 0 016.707 2.707a8.001 8.001 0 1010.586 10.586z"
              />
            </svg>
          </button>
          <a
            href="/login/org"
            onClick={[Function]}
          >
            <button
              className="btn primary lg undefined"
              disabled={false}
              type="button"
            >
               
              Sign In
               
            </button>
          </a>
          <a
            href="/signup/org"
            onClick={[Function]}
          >
            <button
              className="btn transparentbtn lg mr-8"
              disabled={false}
              type="button"
            >
               
              Register an organization
               
            </button>
          </a>
        </div>
        <div
          className="flex px-5 lg:hidden"
        >
          <button
            className="px-3"
            onClick={[Function]}
            type="button"
          >
            <svg
              aria-hidden="true"
              className="w-6 mr-2"
              fill="currentColor"
              viewBox="0 0 20 20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M17.293 13.293A8 8 0 016.707 2.707a8.001 8.001 0 1010.586 10.586z"
              />
            </svg>
          </button>
          <button
            onClick={[Function]}
            type="button"
          >
            <svg
              aria-hidden="true"
              className="w-7 dark:text-dark-text-fill"
              fill="none"
              stroke="currentColor"
              strokeWidth={2}
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M4 6h16M4 12h16M4 18h16"
                strokeLinecap="round"
                strokeLinejoin="round"
              />
            </svg>
          </button>
        </div>
      </div>
      <ul
        className="hidden"
      >
        <li
          className="p-2 w-full mt-2 dark:text-dark-text-fill text-primary"
        >
          <a
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li
          className="p-2 w-full dark:text-dark-text-fill"
        >
          <a
            href="/pricing"
            onClick={[Function]}
          >
            Pricing
          </a>
        </li>
        <li
          className="p-2 w-full dark:text-dark-text-fill"
        >
          <a
            className="w-full"
            href="/product"
            onClick={[Function]}
          >
            Product
          </a>
        </li>
        <li
          className="p-2 w-full dark:text-dark-text-fill mt-6 mb-2 bg-primary text-white rounded-md px-[35%]"
        >
          <a
            className="w-full"
            href="/login/org"
            onClick={[Function]}
          >
             
            Sign In
          </a>
        </li>
        <a
          href="/signup/org"
          onClick={[Function]}
        >
          <button
            className="btn transparentbtn lg mr-8"
            disabled={false}
            type="button"
          >
             
            Register an organization
             
          </button>
        </a>
      </ul>
    </div>
    <div
      className="hero w-full min-h-full flex flex-col justify-center grow items-center py-20"
    >
      <div
        className="grid h-full lg:grid-cols-2 w-full justify-around content-center pb-6"
      >
        <div
          className="flex h-full flex-col items-center justify-start  w-full px-4 py-2"
        >
          <h1
            className="mt-28 sm:mt-20 text-3xl sm:text-4xl md:pl-10 md:text-6xl font-bold text-white dark:text-dark-text-fill font-lexend text-center md:text-left"
          >
            Welcome to DevPulse
          </h1>
          <p
            className="text-2xl md:text-4xl lg:text-3xl mt-8 md:mt-8 lg:mt-8 w-full sm:w-3/4 md:w-4/5 lg:w-full md:px-0 lg:px-8 text-white dark:text-dark-text-fill font-sans text-center"
          >
            The number one platform for
            <em>
              <b>
                 
                managing trainees
                 
              </b>
            </em>
            or
            <em>
              <b>
                 
                students
                 
              </b>
            </em>
            in any
            <em>
              <b>
                 
                ed-tech organization
                 
              </b>
            </em>
          </p>
          <div
            className="w-full text-center justify-center items-center mt-8"
          >
            <a
              href="/signup/org"
              onClick={[Function]}
            >
              <button
                className="btn primary lg mt-12 lg:mt-0 px-8 text-xl font-bold uppercase my-4"
                disabled={false}
                type="button"
              >
                 
                Get Started
                 
              </button>
            </a>
          </div>
        </div>
      </div>
    </div>
    <div
      className="w-full bg-primary dark:bg-dark-bg text-gray-300 mt-auto dark:bg-opacity-75 bg-opacity-100"
    >
      <div
        className="px-2 flex flex-col lg:flex-row justify-between items-center w-full h-full"
      >
        <div
          className="flex flex-col lg:flex-row items-center py-5"
        >
          <div
            className="flex flex-col mr-12 lg:mr-0"
          >
            <div
              className="flex mb-2"
            >
              <img
                alt="logo"
                className="mr-2 lg:mr-0"
                src={Object {}}
              />
              <h1
                className="text-3xl font-bold text-white dark:text-dark-text-fill"
                data-testid="pulse"
              >
                PULSE
              </h1>
            </div>
            <div
              className="flex border-none justify-around items-center ml-6 mt-4 cursor-pointer"
            >
              <svg
                className=" fa-xs "
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 512 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M504 256C504 119 393 8 256 8S8 119 8 256c0 123.78 90.69 226.38 209.25 245V327.69h-63V256h63v-54.64c0-62.15 37-96.48 93.67-96.48 27.14 0 55.52 4.84 55.52 4.84v61h-31.28c-30.8 0-40.41 19.12-40.41 38.73V256h68.78l-11 71.69h-57.78V501C413.31 482.38 504 379.78 504 256z"
                />
              </svg>
              <svg
                className=" "
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 448 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
                />
              </svg>
              <svg
                className="mr-1 "
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 512 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
                />
              </svg>
              <svg
                className="mr-1 "
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 448 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
                />
              </svg>
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 512 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm115.7 272l-176 101c-15.8 8.8-35.7-2.5-35.7-21V152c0-18.4 19.8-29.8 35.7-21l176 107c16.4 9.2 16.4 32.9 0 42z"
                />
              </svg>
            </div>
          </div>
          <div
            className="flex flex-col py-4 lg:py-0 lg:flex-row"
          >
            <div
              className="flex py-4"
            >
              <ul
                className="lg:flex lg:flex-col mr-[8vh] md:mr-[16vh] lg:mr-0 lg:ml-28 cursor-pointer"
              >
                <li
                  className="py-2 text-sm font-bold"
                >
                  Dev Pulse
                </li>
                <li
                  className="py-2 text-xs"
                >
                  About us
                </li>
                <li
                  className="py-2 text-xs"
                >
                  Contact us
                </li>
              </ul>
              <ul
                className="lg:flex lg:flex-col ml-[8vh] md:ml-[16vh] lg:ml-28 cursor-pointer"
              >
                <li
                  className="py-2 text-sm font-bold"
                >
                  Product
                </li>
                <li
                  className="py-2 text-xs"
                >
                  Features
                </li>
                <li
                  className="py-2 text-xs"
                >
                  Integrations
                </li>
              </ul>
            </div>
            <div
              className="flex py-4"
            >
              <ul
                className="lg:flex lg:flex-col mr-[8vh] md:mr-[16vh] lg:mr-0 lg:ml-28 cursor-pointer"
              >
                <li
                  className="py-2 text-sm font-bold"
                >
                  Resources
                </li>
                <li
                  className="py-2 text-xs"
                >
                  Community
                </li>
                <li
                  className="py-2 text-xs"
                >
                  Help Center
                </li>
              </ul>
              <ul
                className="lg:flex lg:flex-col ml-[8vh] md:ml-[16vh] lg:ml-28 cursor-pointer"
              >
                <li
                  className="py-2 text-xs"
                >
                  Terms and conditions
                </li>
                <li
                  className="py-2 text-xs"
                >
                  Privacy and Policies
                </li>
                <select
                  className="bg-dark-bg mt-2 outline rounded-md px-2 py-1 text-white dark:text-dark-text-fill dark:bg-dark-bg "
                  data-testid="lanChange"
                  defaultValue="en-US"
                  name="language"
                  onChange={[Function]}
                >
                  <option
                    value="en"
                  >
                    English
                  </option>
                  <option
                    value="kn"
                  >
                    Kinyarwanda
                  </option>
                  <option
                    value="fr"
                  >
                    Français
                  </option>
                </select>
              </ul>
            </div>
          </div>
        </div>
        <div
          className=" lg:flex"
        >
          <span
            className="px-4 lg:py-3 cursor-pointer text-lg"
          >
            © 
            2023
             Pulse Technologies
          </span>
        </div>
      </div>
    </div>
  </div>
  <div
    className="w-full bg-primary dark:bg-dark-bg text-gray-300 mt-auto undefined"
  >
    <div
      className="px-2 flex flex-col lg:flex-row justify-between items-center w-full h-full"
    >
      <div
        className="flex flex-col lg:flex-row items-center py-5"
      >
        <div
          className="flex flex-col mr-12 lg:mr-0"
        >
          <div
            className="flex mb-2"
          >
            <img
              alt="logo"
              className="mr-2 lg:mr-0"
              src={Object {}}
            />
            <h1
              className="text-3xl font-bold text-white dark:text-dark-text-fill"
              data-testid="pulse"
            >
              PULSE
            </h1>
          </div>
          <div
            className="flex border-none justify-around items-center ml-6 mt-4 cursor-pointer"
          >
            <svg
              className=" fa-xs "
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                Object {
                  "color": undefined,
                }
              }
              viewBox="0 0 512 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M504 256C504 119 393 8 256 8S8 119 8 256c0 123.78 90.69 226.38 209.25 245V327.69h-63V256h63v-54.64c0-62.15 37-96.48 93.67-96.48 27.14 0 55.52 4.84 55.52 4.84v61h-31.28c-30.8 0-40.41 19.12-40.41 38.73V256h68.78l-11 71.69h-57.78V501C413.31 482.38 504 379.78 504 256z"
              />
            </svg>
            <svg
              className=" "
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                Object {
                  "color": undefined,
                }
              }
              viewBox="0 0 448 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
              />
            </svg>
            <svg
              className="mr-1 "
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                Object {
                  "color": undefined,
                }
              }
              viewBox="0 0 512 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
              />
            </svg>
            <svg
              className="mr-1 "
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                Object {
                  "color": undefined,
                }
              }
              viewBox="0 0 448 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
              />
            </svg>
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                Object {
                  "color": undefined,
                }
              }
              viewBox="0 0 512 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm115.7 272l-176 101c-15.8 8.8-35.7-2.5-35.7-21V152c0-18.4 19.8-29.8 35.7-21l176 107c16.4 9.2 16.4 32.9 0 42z"
              />
            </svg>
          </div>
        </div>
        <div
          className="flex flex-col py-4 lg:py-0 lg:flex-row"
        >
          <div
            className="flex py-4"
          >
            <ul
              className="lg:flex lg:flex-col mr-[8vh] md:mr-[16vh] lg:mr-0 lg:ml-28 cursor-pointer"
            >
              <li
                className="py-2 text-sm font-bold"
              >
                Dev Pulse
              </li>
              <li
                className="py-2 text-xs"
              >
                About us
              </li>
              <li
                className="py-2 text-xs"
              >
                Contact us
              </li>
            </ul>
            <ul
              className="lg:flex lg:flex-col ml-[8vh] md:ml-[16vh] lg:ml-28 cursor-pointer"
            >
              <li
                className="py-2 text-sm font-bold"
              >
                Product
              </li>
              <li
                className="py-2 text-xs"
              >
                Features
              </li>
              <li
                className="py-2 text-xs"
              >
                Integrations
              </li>
            </ul>
          </div>
          <div
            className="flex py-4"
          >
            <ul
              className="lg:flex lg:flex-col mr-[8vh] md:mr-[16vh] lg:mr-0 lg:ml-28 cursor-pointer"
            >
              <li
                className="py-2 text-sm font-bold"
              >
                Resources
              </li>
              <li
                className="py-2 text-xs"
              >
                Community
              </li>
              <li
                className="py-2 text-xs"
              >
                Help Center
              </li>
            </ul>
            <ul
              className="lg:flex lg:flex-col ml-[8vh] md:ml-[16vh] lg:ml-28 cursor-pointer"
            >
              <li
                className="py-2 text-xs"
              >
                Terms and conditions
              </li>
              <li
                className="py-2 text-xs"
              >
                Privacy and Policies
              </li>
              <select
                className="bg-dark-bg mt-2 outline rounded-md px-2 py-1 text-white dark:text-dark-text-fill dark:bg-dark-bg "
                data-testid="lanChange"
                defaultValue="en-US"
                name="language"
                onChange={[Function]}
              >
                <option
                  value="en"
                >
                  English
                </option>
                <option
                  value="kn"
                >
                  Kinyarwanda
                </option>
                <option
                  value="fr"
                >
                  Français
                </option>
              </select>
            </ul>
          </div>
        </div>
      </div>
      <div
        className=" lg:flex"
      >
        <span
          className="px-4 lg:py-3 cursor-pointer text-lg"
        >
          © 
          2023
           Pulse Technologies
        </span>
      </div>
    </div>
  </div>
</div>
`;
